.github/workflows/tests.yml

```diff
@@ -59,18 +70,40 @@ jobs:
         ]
     steps:
       - uses: actions/checkout@v2
-      - name: Install toolchain
-        uses: actions-rs/toolchain@v1
+      - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
           target: ${{ matrix.target }}
           toolchain: stable
-          override: true
-      # update is needed to fix the 404 error on install, see:
-      # https://github.com/actions/virtual-environments/issues/675
-      - run: sudo apt-get update
-      - run: sudo apt-get install gcc-multilib
-      - run: cargo test --target ${{ matrix.target }}
+      - name: Install multilib
+        # update is needed to fix the 404 error on install, see:
+        # https://github.com/actions/virtual-environments/issues/675
+        run: |
+          sudo apt-get update
+          sudo apt-get install gcc-multilib
+      - run: cargo test --target=${{ matrix.target }} --features=std
```

Comment:
Let me do a quick check for which one is faster

Reply:
It looks like installing via `actions-rs/toolchain` is as fast as installing via `rustup` directly.

Or did you mean that we should:
- Install all three targets
- Run tests on all three targets

In a single job?

Reply:
Yes, I meant merging them into a single job.

Reply:
I think the overall solution to this would be toolchain caching, so that we don't download the same stable toolchain twice during our CI runs.

It looks like the actions-rs folks are working on this: https://github.com/actions-rs/meta/issues/21, so hopefully we will get this for free when they update `actions-rs/toolchain`.

Reply:
Hmm it looks like there's also: https://github.com/actions-rs/toolchain/issues/54, so I'll just add toolchain caching to the TODO bug.